.taskcluster.yml

```diff
@@ -1,25 +1,21 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
+---
 version: 1
+reporting: checks-v1
 policy:
   pullRequests: public
 tasks:
   $let:
+    taskgraph:
+      branch: taskgraph
+      revision: f949fb56d96b9509b071ba55853f2d60e8dbbf57
+    trustDomain: glean
```

Comment:
We'll need to ramp-up workers for this. I'll take care of this in https://bugzilla.mozilla.org/show_bug.cgi?id=1628403